reportportal_client/helpers.py

```diff
@@ -18,6 +18,7 @@
 import uuid
 import warnings
 from platform import machine, processor, system
+from aenum import PY3
```

Comment:
Please use `six` for this test, as here:
https://github.com/reportportal/client-Python/blob/develop/reportportal_client/logs/__init__.py#L18